analyzer/codechecker_analyzer/analyzers/config_handler.py

```diff
@@ -73,6 +75,23 @@ def analyzer_plugins(self):
                             and f.endswith(".so")]
         return analyzer_plugins
 
+    def get_version(self, env=None):
+        """ Get analyzer version information. """
+        version = [self.analyzer_binary, ' --version']
+        try:
+            output = subprocess.check_output(shlex.split(' '.join(version)),
```

Comment:
```suggestion
            output = subprocess.check_output(version),
```
Do we need the shlex split here? `version` is already a list of the parts of the command it should be enough.